### DISCUSSION TITLE
Transfer asdf-cargo-make, asdf-yamlfmt and asdf-zigmod

### DISCUSSION THREAD

**Discussion by kachick** [mise-plugins/.github#13] [Transfer Request]
Can I transfer my three asdf plugins into this organization?

https://github.com/kachick/asdf-cargo-make
https://github.com/kachick/asdf-yamlfmt
https://github.com/kachick/asdf-zigmod

I agree to release my commit rights as written [here](https://github.com/mise-plugins/.github/blob/3e9352880e88fe2962a412b486ee1f0a9e65523a/profile/README.md?plain=1#L87-L89).

**Comment by jdx** [comment]
of course!

**Comment by kachick** [comment]
Thank you for the approval. I have updated the LICENSE files and sent the transfer requests.

https://github.com/kachick/asdf-cargo-make/commit/475f0d77e836f57df7bf22354b5f6daa58208c2d
https://github.com/kachick/asdf-yamlfmt/commit/5f36f2ed0c021f5170a64f7b2e606fc474e33ba9
https://github.com/kachick/asdf-zigmod/commit/07d8e84fa0ea51de2aa3256152313d05c7b0b081

I have used renovatebot to update the .tool-versions file. After transferring repositories, it may have stopped.
